mistaken-pull-closer.yml

```diff
@@ -0,0 +1,17 @@
+# The JSONPath filter expression used to identify which PRs to close.
+# The data filtered is the pull request data along with other metadata passed in
+# by probot.
+# See http://goessner.net/articles/JsonPath/
+# `true` will close all PRs.
+filters:
+    - true
+
+# The message to post to the closed PR.
+commentBody: |
+  Thanks for your contribution!  Unfortunately, we don't use GitHub pull
+  requests to manage code contributions to this repository.  Instead, please
```

Comment:
It might help to call out this is a mirror as it explains why PR's aren't accepted.

> Unfortunately, this GitHub repository is a mirror of [http://source.andoird.com] and cannot be used for accepting contributions. Instead, please see ...

Reply:
Done, thanks.